app/models/address.rb

```diff
@@ -1,5 +1,4 @@
 class Address < ApplicationRecord
   validates :zip_code, :state, presence: true
```

Comment:
not sure how you can validate state if state isn't actually defined here? I think the belongs to was right... would use a unit test to confirm the relationship is setup correctly